benchmarks/scripts/plot_avg_benchmarks_over_time.py

```diff
@@ -66,7 +68,8 @@
 
 # Filter on first occurrence of each compiler version based on the date
 df_dates = df_dates[df_dates["date"].isin(unique_dates)]
-
+if SAVE_LEGACY:
+    df_dates.to_csv("./legacy_timing_results.csv", index=False)
```

Comment:
When would we want to set `SAVE_LEGACY = True`?

Reply:
I did this just once and manually imported the results here https://github.com/unitaryfoundation/ucc-bench/blob/main/results/ucc-benchmarks-8-core-U22.04/compilation_benchmarks/legacy_timing_results.csv.

We shouldn't need to redo that, so I could probably just drop this change. But was convenient enough to keep and I expect us to remove these benchmark scripts once we flip over to `ucc-bench`.